exonum/src/lib.rs

```diff
@@ -21,6 +21,7 @@
 #![cfg_attr(feature = "flame_profile", feature(plugin, custom_attribute))]
 #![cfg_attr(feature = "flame_profile", plugin(exonum_flamer))]
 #![cfg_attr(feature = "long_benchmarks", feature(test))]
+#![cfg_attr(feature = "cargo-clippy", deny(unseparated_literal_suffix))]
```

Comment:
I prefer to enable all pedantic warnings and disable some of them instead of enabling individual lints.